src/main/pages/che-7/end-user-guide/proc_converting-a-che-6-workspace-to-a-basic-che-7-devfile.adoc

```diff
@@ -46,10 +47,33 @@ CentOS nodejs
 
 | Python, +
 Python with Theia IDE
-| Python with Theia IDE
+| Python
+
+| Eclipse Vert.x
+| Java Vert.x
+
+| PHP
+| PHP Simple
+
+| Spring Boot
+| Java Spring Boot
 |===
 
-. Add the required number of blank projects.
-. Change the required *devfile* fields.
+.. By default, the example project is added to the workspace. The default project can be removed by clicking the *Remove* button:
++
+image::workspaces/remove-default-project.png[link="{imagesdir}/workspaces/remove-default-project.png"]
+
+.. One can import a custom project that was used in Che 6 workspace by clicking the *Add or Import Project* and selecting *Git* or *Github* options:
++
+image::workspaces/add-custom-project.png[link="{imagesdir}/workspaces/add-custom-project.png"]
+
+.. Once the project is imported, various commands could be added to the devfile e.g `run`, `build`, `test`. Those commands will be later accessible from the IDE once the workspace is started. Custom commands and other devfile components can be added in the *Devfile* configuration.
+
+.. Select the *Create & Proceed Editing* button. 
++
+image::workspaces/create-and-edit.png[link="{imagesdir}/workspaces/create-and-edit.png"]
++
+Select the *Devfile* tab in order to update the configuration. See the link:https://redhat-developer.github.io/devfile/devfile[Devfile specification] for the detailed information about the supported properties and attributes.
 +
-image::workspaces/devfile-fields.png[link="{imagesdir}/workspaces/devfile-fields.png"]
+image::workspaces/update-devfile-configuration.png[link="{imagesdir}/workspaces/update-devfile-configuration.png"]
+.. Once the *Devfile* configuration setup is finished, press the *Open* button in order to start a newly created Che 7 workspace.
```

Comment:
Something that may be helpful to mention is that machine `servers` in workspace configs can be specified as devfile components `endpoints` and `installers` as components of type `chePlugin`.

Reply:
@l0rd @ibuziuk I'm not much familiar with that so I'm not sure I'm able to document that properly. Could one of you create a suggestion that I could merge please?

Reply:
I have added a short suggestion. That's not much but better than nothing ;-)